Update Overview_Training_certification.md (#3749)
SkylineCommunications/dataminer-docs: dataminer-overview/Training_courses_and_certification/Overview_Training_certification.md

> [!TIP]
> For more information about the different certifications and their process, please visit [DataMiner Certification](https://community.dataminer.services/learning/certification/).

## DataMiner Fundamentals exam guide

Are you preparing to take the DataMiner Fundamentals exam? Below you can find all the information you need, including the certification process, the exam format, the scoring criteria, and tips on how to prepare for the exam.

### Certification process

As soon as the ordering process is finalized and your certification process can be started, you will receive an automated email from Skyline Academy with information about the process and the deadline.

You will also receive an email with your user ID and password to access the training DMA in order to practice and to take the exam.

To book an appointment, you will need to use the booking link that you will find in the email you received with information about the start of the certification process.

> [!NOTE]
>

> > - The certification process is valid for 3 months, unless otherwise agreed.

Reviewer: Do you mean that the maximum duration of the certification process is 3 months, unless otherwise agreed?

Author: The candidate has max. 3 months to finalize the process (pass the exam incl. retakes). Unless agreed otherwise because some organizations want their employees to pass the exam within 1 month after the process started.

> - By default, the training DMA will be used to conduct the exam.
> - You are allowed to use DataMiner Docs during the exam.
> - If you do not pass the exam, you have one free retake.
> - Exams are recorded to ensure the integrity of the process and to provide a reference for review if needed.

### Preparation tips

To prepare effectively for the exam:

- Study materials: review the [recommended course](https://community.dataminer.services/courses/dataminer-administrator/) and [online resources](https://docs.dataminer.services/).
- Practice questions: use the [quiz](https://community.dataminer.services/courses/dataminer-administrator/quizzes/dataminer-administrator/) at the end of the course to practice. The quiz randomly selects 10 questions out of a pool of questions.
- Schedule check-ins with your examiner via the booking page if you have any questions.

### Examples questions

- What technology is used to manage and execute backup schedules?
- Is it possible to test queries within Cube? If so, where can you find this feature?
- How would you create a sample scheduled task with specific details such as name, schedule, time of execution, and actions?

> - How do you define the relationship between objects in Microsoft Visio with DataMiner shapes for elements, views, etc.?

Reviewer: This seems a bit odd. Is this what you mean?
```suggestion
- How do you define the relationship between shapes in Microsoft Visio and DataMiner objects such as elements, views, etc.?
```

Author: Will need to check with one of the trainers. 
The questions  they use in the exam plan is:
How do you define the relationship between objects in MS Visio with DataMiner shapes for elements, views, etc....

Answer: 
You use shape data in visio. The labels and value fields need to be used.

Reviewer: If that is the answer, I think the way I rephrased it is indeed better, because strictly speaking there is no such thing as "DataMiner shapes", but instead there are Visio shapes referring to DataMiner objects.

Author: thanks - I've updated the question in the exam plan as well

> For more information about the different certifications and their process, please visit [DataMiner Certification](https://community.dataminer.services/learning/certification/).

## DataMiner Fundamentals exam guide

Are you preparing to take the DataMiner Fundamentals exam? Below you can find all the information you need, including the certification process, the exam format, the scoring criteria, and tips on how to prepare for the exam.

### Certification process

As soon as the ordering process is finalized and your certification process can be started, you will receive an automated email from Skyline Academy with information about the process and the deadline.

You will also receive an email with your user ID and password to access the training DMA in order to practice and to take the exam.

> To book an appointment, you will need to use the booking link that you will find in the email you received with information about the start of the certification process.

Author: I also want to add the info that candidates receive monthly reminders by email so don't forget about the process. 
20, 10, 5 days before the deadline they also receive emails to inform them that their process will expire soon. Once expired they also receive an email. 
This information is the same for the configurator process

> 

Author: I also want to add that candidates receive automatic reminders via email so they don't forget about their process.
20, 10, 5 days before the expiration date of their process they also receive an email to inform them that their process will expire soon. Once expired they are also notified via email. 
This info is the same for the configurator process.
